spec/features/course_page_spec.rb

```diff
@@ -224,6 +192,24 @@ def js_visit(path, count=3)
       end
       expect(Course.last.passcode).to eq(previous_passcode)
     end
+
+    context 'allows edit in home_wiki_id' do
```

Comment:
The `context` text should refer to the setup being done in this block, which in this case is the 'before'.

Then the `it` block text should refer to what you're actually testing.

And together, they should form a complete-ish, natural language description of the test.